packages/dev/core/src/Meshes/GreasedLine/greasedLineRibbonMesh.ts

```diff
@@ -237,7 +237,7 @@ export class GreasedLineRibbonMesh extends GreasedLineBaseMesh {
         }
 
         const v: number[] = [1, 0, numOfPaths];
-        const doubleSided = options.ribbonOptions?.facesMode === GreasedLineRibbonFacesMode.FACES_MODE_DOUBLE_SIDED ?? false;
+        const doubleSided = options.ribbonOptions?.facesMode === GreasedLineRibbonFacesMode.FACES_MODE_DOUBLE_SIDED || false;

```

Comment:
Why is there a ?? or a || in this case? FACES_MODE_DOUBLE_SIDED is an entry in a const enum with the value 2, so how would the || ever do anything?

Reply:
I didn't want to fix code issues, just make typescript compile it. but you are totally right, this can be removed.